tests/PlusPull/Commands/CheckTest.php

```diff
@@ -143,19 +143,28 @@ public function testExecute(
                 'password' => $password,
                 'token'    => $token,
             ),
-            'repository' => array(
-                'name' => 'test-repo',
-                'username' => 'test-owner',
-                'status' => true,
-                'required' => $required,
-                'whitelist' => $whitelist,
-                'labels' => array(
-                    array(
-                        'name' => 'blocked',
-                        'color' => 'eb6420',
-                        'hook' => '[B]',
+            'repositories' => array(
```

Comment:
@christofdamian I'm not sure if that change should be accepted, but I just saw duplicating `testExecute` as an alternative, and I had some worries about adding too much overhead to the test run.